packages/block-editor/src/components/media-replace-flow/test/index.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * External dependencies
+ */
+import { noop } from 'lodash';
```

Comment:
We can also get rid of `noop` with something like - `const noop = () => {};`

Reply:
A good idea which I followed 👌🏻